### PR TITLE
Added new MLNX HWSKU ACS-MSN4410

### PR DESCRIPTION
**Pull request by yvolynets-mlnx**
Signed-off-by: Yuriy Volynets <yuriyv@nvidia.com>

### Description of PR

Summary: Updated mellanox_data.py with new HWSKU - "ACS-MSN4410"
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [x] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

#### What is the motivation for this PR?
To extend list of supported MLNX HWSKUs.

### CODE REVIEW

File: tests/common/mellanox_data.py [hunk 1]
```diff
@@ -559,6 +559,59 @@
                 "number": 1
             }
         }
+    },
+    "ACS-MSN4410": {
```
Comment (wangxin) [inline tests/common/mellanox_data.py:563]: Why use hwsku instead of platform like the others here?

**Review by wangxin — COMMENTED** [review]
I am not sure if this is complete. I noticed that for each of a Mellanox switch model, there are also other data in files like:
* `ansible/group_vars/sonic/sku-sensors-data.yml`
*  `ansible/group_vars/sonic/variables`
*  `tests/platform_tests/mellanox/minimum_table.py`

File: tests/common/mellanox_data.py (hunk 1 above)
Comment (yvolynets-mlnx) [reply tests/common/mellanox_data.py]: Fixed, thank you.

**Comment by wangxin** [comment]
@yvolynets-mlnx For a new HWSKU, I think these files need to be updated as well, right?

* `ansible/group_vars/sonic/sku-sensors-data.yml`
* `ansible/group_vars/sonic/variables`
* `tests/platform_tests/mellanox/minimum_table.py`

**Comment by yvolynets-mlnx** [comment]
> 
> 
> @yvolynets-mlnx For a new HWSKU, I think these files need to be updated as well, right?
> 
>     * `ansible/group_vars/sonic/sku-sensors-data.yml`
> 
>     * `ansible/group_vars/sonic/variables`
> 
>     * `tests/platform_tests/mellanox/minimum_table.py`

Fixed. Please take a look.

**Review by wangxin — APPROVED** [review]

**Pull request merged**